KopernicusExamples/Creating New Bodies/GasGiant/GasPlanet.cfg

```diff
@@ -1,9 +1,23 @@
-@Kopernicus:AFTER[Kopernicus]
+@Kopernicus:AFTER[Kopernicus]:FOR[Bastion]
 {
     Body
     {
         name = Bastion
-
+		cacheFile = KopernicusExamples/Creating New Bodies/GasGiant/Cache/Bastion.bin
+		
+		Debug
+		{
+			exportMesh = true
+			update = true
+			showSOI = false
+		}
+		
+		Template
```

Comment:
I think Bastion was intended as an example for making Gas Planets without a template.. It doesn't really matter since we have the Kopernicus planet for that, but maybe you could add a comment that the Template node is optional, assuming that it still works without one?